pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -245,6 +247,14 @@ func (alloc *Action) allocateResourceForTasksWithTopology(tasks *util.PriorityQu
 			break
 		}
 		for _, hyperNodeName := range ssn.HyperNodesListByTier[tier] {
+			if jobHyperNode == "" {
+				// job first scheduler
```

Comment:
-> `//job is scheduled for the first time`

Reply:
done